lib/utils.js

```diff
@@ -21,3 +21,13 @@ export function deprecated (fn, message) {
 
   return newFn
 }
+
+export function printAndExit (message, code = -1) {
```

Comment:
@nkzawa I don't think there's any different. For errors it should be non-zero code.
Change it to 1 if you like.